src/alloy/AlloyInterface.java

```diff
@@ -27,19 +27,12 @@
     private static final A4Reporter reporter = new A4Reporter();
     private static final A4Options options = new A4Options();
 
-    public static CompModule compile(String modelPath) {
-        try {
-            return CompUtil.parseEverything_fromFile(reporter, null, modelPath);
-        } catch (Err e) {}
-
-        return null;
+    public static CompModule compile(String modelPath) throws Err {
+        return CompUtil.parseEverything_fromFile(reporter, null, modelPath);
     }
 
-    public static A4Solution run(CompModule module) {
+    public static A4Solution run(CompModule module) throws Err {
         List<Command> commands = module.getAllCommands();
-        if (commands.isEmpty()) {
```

Comment:
While this case doesn't occur due to our injected command, this class is independent from the rest of the code, and I feel it shouldn't make any assumptions about its use.

Maybe we can throw `Err` if the list is empty?

Reply:
Actually, I don't think that should be a concern for us at all due to [the default command](https://github.com/AlloyTools/org.alloytools.alloy/blob/master/org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/parser/CompUtil.java#L452) that is always injected into `CompModule`s.